<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Двухпросмотровый ассемблер</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="sourceCodeLabel">
      <property name="text">
       <string>Исходный текст</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="sourceCodeTextEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="commandsLabel">
      <property name="text">
       <string>Таблица кодов операций</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="commandsTextEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="firstPassLabel">
      <property name="text">
       <string>Вспомогательная таблица</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="firstPassTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="tsiLabel">
      <property name="text">
       <string>Таблица символических имен</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="tsiTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="firstPassErrorsLabel">
      <property name="text">
       <string>Ошибки первого прохода</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="firstPassErrorsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="secondPassLabel">
      <property name="text">
       <string>Двоичный код</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="secondPassTextEdit"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="secondPassErrorsLabel">
      <property name="text">
       <string>Ошибки второго прохода</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTextEdit" name="secondPassErrorsTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="firstPassButton">
        <property name="text">
         <string>Первый проход</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="secondPassButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Второй проход</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
